lightning/src/ln/channelmanager.rs

```diff
@@ -2731,7 +2731,12 @@ where
 		if onion_utils::route_size_insane(&onion_payloads) {
```

Comment:
Is the panic mentioned in the PR description reachable? I thought this check would prevent it